rflx/pyrflx/typevalue.py

```diff
@@ -717,10 +713,13 @@ def _preset_fields(self, fld: str) -> None:
             self._last_field = nxt
             nxt = self._next_field(nxt)
 
-    def get(self, fld: str) -> Any:
-        if fld not in self.valid_fields:
-            raise ValueError(f"field {fld} not valid")
-        return self._fields[fld].typeval.value
+    def get(self, field_name: str) -> Any:
+        if field_name not in self.valid_fields:
+            raise ValueError(f"field {field_name} not valid")
+        field = self._fields[field_name]
+        if isinstance(field.typeval, OpaqueValue) and field.typeval.nested_message is not None:
+            return field.typeval.nested_message
+        return self._fields[field_name].typeval.value
```

Comment:
`get` should not return `Any`. With the current signature a user of this method does not have to check the type of the result and could just do anything with it.

https://mypy.readthedocs.io/en/stable/dynamic_typing.html?#operations-on-any-values

Reply:
Fixed by bfda76d